src/components/Treeview/EnrichedTreeItem.js

```diff
@@ -45,18 +45,18 @@ const useStyles = makeStyles(styles);
 const EnrichedTreeItem = ({
     label,
     id,
-    fetchChildrenData, // fetchChildrenData(id)
-    expanded,
-    toggleOnLabelClick,
-    onLabelClick,
-    data, // additional data that can be passed up to the parent (eg org unit details)
-    withCheckbox,
-    ticked,
-    parentsTicked,
-    scrollIntoView,
-    allowSelection,
+    fetchChildrenData = () => { },
```

Comment:
```suggestion
    fetchChildrenData = () => { }, // fetchChildrenData(id)
```

This will be useful when converting to TS